implementations/rust/ockam/ockam_command/src/authenticated.rs

```diff
@@ -43,7 +43,9 @@ pub enum AuthenticatedSubcommand {
 
 impl AuthenticatedCommand {
     pub fn run(c: AuthenticatedCommand) {
-        embedded_node(run_impl, c.subcommand)
+        if let Err(e) = embedded_node(run_impl, c.subcommand) {
+            eprintln!("Ockam node failed: {:?}", e,);
+        }
```

Comment:
Hey, shouldn't this return `exitcode`? Sorry if I'm missing something.

Reply:
Hey! This is just a workaround. As soon as we refactor that command (and all the others) to use the new methods from this PR, most exit codes will be handled automatically (including the one you mentioned).